resources/nautilus.toml

```diff
@@ -30,5 +30,11 @@ pbin = 2
 vflip = false
 hflip = true
 
+[device.kinetix.line_read_times]
+0 = 3.53125
```

Comment:
instead of using numbers as keys here you could use the names for each of the modes

```toml
[device.kinetix.line_read_times]
dynamic_range = 3.75
speed = 0.625
sensitivity = 3.53125
sub_electron = 60.1
```